lib/agenda/jobs.ts

```diff
@@ -11,9 +12,9 @@ import { createJob } from '../utils';
  * @param number of documents to skip in MongoDB
  * @returns resolves when fails or passes
  */
-export const jobs = async function(this: Agenda, query = {}, sort = {}, limit = 0, skip = 0) {
+export const jobs = async function(this: Agenda, query: FilterQuery<any> = {}, sort = {}, limit = 0, skip = 0) {
   const result = await this._collection
-    .find(query)
+    .find(query) // eslint-disable-line unicorn/no-fn-reference-in-iterator
```

Comment:
Just for reference for anyone reading, since I was curious myself what was happening here: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/755